qa/workunits/ceph-helpers.sh

```diff
@@ -1452,13 +1472,13 @@ function main() {
 
     local code
     if run $dir "$@" ; then
-        code=0
+        teardown $dir || return 1
+        return 0
     else
         display_logs $dir
-        code=1
+        teardown_error $dir
+        return 1
     fi
-    teardown $dir || return 1
```

Comment:
@wjwithagen can we leave this change for another changeset? this changes the behaviour of current tests.